### PR TITLE
Create whisperv6 patch file

### LINKED ISSUE #636: WhisperV6 as a Geth patch
# Problem

We need first implementation of whisperV6 to make it possible to implement Status patcher to it and start to use it.
Our current version of whisperV6 is a just clone of whisperV5. We need to get updates from upstream.

# Implementation

Move whisperV6's code into our Geth version. Make it as a temporary patch.

# Acceptance Criteria

Status-go compiles successfully, tests in whisperV6 packages are passed.

# Notes

That should be temparary patch until we rebase to Geth 1.8+

Part of #634

### PR DESCRIPTION
**Pull request by pedropombeiro**
This PR creates a patch file in geth-patches to import whisperv6 (taken from the to the `2b4c7e9` commit in upstream `master`). It depends on https://github.com/status-im/status-go/pull/647.

Closes #636

### CODE REVIEW

**Review by JekaMas — APPROVED** [review]
LGTM

File: _assets/patches/geth/README.md [hunk 1]
```diff
@@ -27,18 +30,21 @@ When a new stable release of `go-ethereum` comes out, we need to upgrade our for
 #### I. In our fork at /status-im/go-ethereum.
 
 1. Remove the local `develop` branch.
+
 ```bash
 git branch -D develop
 ```
 
-2. Pull upstream release branch into `develop` branch.
+1. Pull upstream release branch into `develop` branch.
```
Comment (b00ris) [inline _assets/patches/geth/README.md]: Why did you change it?

Comment (pedropombeiro) [reply _assets/patches/geth/README.md]: Because the best practice in markdown files is to always use `1.`. The markdown renderer will take care of putting the right numbers, and that way you can insert new items into a list and not have to care about manually renumbering paragraphs.

**Review by b00ris — APPROVED** [review]
